scout/adapter/mongo/case.py

```diff
@@ -876,8 +876,10 @@ def load_case(self, config_data, update=False, keep_actions=True):
             if case_obj["status"] in ["active", "archived"]:
                 case_obj["status"] = "inactive"
 
+            self.case_variants_count(
```

Comment:
This will update the number of variants by type in the case document, but it won't modify the date of analysis